src/pudl/transform/eia861.py

```diff
@@ -1663,6 +1786,131 @@ def utility_data(tfr_dfs):
         dict: A dictionary of transformed EIA 861 dataframes, keyed by table name.
 
     """
+    idx_cols = [
+        'utility_id_eia',
+        'state',
+        'report_date',
+        'nerc_region'
+    ]
+
+    # Pre-tidy clean specific to operational data table
+    raw_ud = (
+        tfr_dfs["utility_data_eia861"].copy()
+        .query("utility_id_eia not in [88888]")
+
+    )
+
+    ###########################################################################
+    # Transform Data Round 1:
+    # * Clean NERC region col
+    ###########################################################################
+
+    transformed_ud = _clean_nerc(raw_ud, idx_cols)
+
+    # Establish columns that are nerc regions vs. rtos
+    nerc_cols = [col for col in raw_ud if 'nerc_region_operation' in col]
+    rto_cols = [col for col in raw_ud if 'rto_operation' in col]
```

Comment:
neat! i didn't know you could do a list comprehension on column names w/o using `dataframe.columns`